### PR TITLE
nature-banner / springer-banner: Update settings files

### PR DESCRIPTION
**Pull request by davidpauljunior**
- Renames the settings files to make it easier to override (this is a temporary move until the merging logic puts the global settings into a separate folder)
- Updates the values to use the brand functions

### CODE REVIEW

File: toolkits/nature/packages/nature-banner/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 0.3.0 (2019-01-17)
```
Comment (ricky-lau) [inline toolkits/nature/packages/nature-banner/HISTORY.md]: date needs updating

File: toolkits/springer/packages/springer-banner/HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# History
+
+## 0.2.0 (2019-01-17)
```
Comment (ricky-lau) [inline toolkits/springer/packages/springer-banner/HISTORY.md]: date needs updating

File: toolkits/nature/packages/nature-banner/HISTORY.md (hunk 1 above)
Comment (davidpauljunior) [reply toolkits/nature/packages/nature-banner/HISTORY.md]: does it...

Comment (davidpauljunior) [reply toolkits/nature/packages/nature-banner/HISTORY.md]: Oh, I'm in the wrong year and have been for a while.

Comment (ricky-lau) [reply toolkits/nature/packages/nature-banner/HISTORY.md]: just noticed that now too 😆

**Pull request merged**